Dockerfile

```diff
@@ -1,23 +1,28 @@
-# Fetch the dependencies
-FROM golang:1.15-alpine AS builder
-
-RUN apk add --update ca-certificates git gcc g++ libc-dev
-WORKDIR /src/
-
-ENV GO111MODULE=on
-
-COPY go.mod .
-COPY go.sum .
-
+FROM alpine AS base
+RUN apk add --no-cache git terraform wget make && \
+    wget -q https://github.com/goreleaser/goreleaser/releases/download/v1.21.2/goreleaser_1.21.2_x86_64.apk  && \
+    wget -q https://go.dev/dl/go1.21.1.linux-amd64.tar.gz && \
```

Comment:
FYI, we are still using go 1.19 because JFrog Pipeline only support 1.19 runtime.